### ISSUE TITLE
Setting SWO speed above 50 MHz on J-Trace

### ISSUE THREAD

**Issue by graeme-hattan** [square/pylink#91]
According to Segger, the J-Trace PRO models are capable of SWO speeds up to [100 MHz](https://www.segger.com/products/debug-probes/j-trace/models/model-overview/).

However, when I call jlink.swo_start(100_000_000), I the following dialog pops up:

![image](https://user-images.githubusercontent.com/1633586/113297684-d3e9fc00-92f2-11eb-8ba7-6a0344e4bc9e.png)

Is there a ways to enable speeds above 50 MHz?

**Comment by hkpeprah** [comment]
Are you sure you have the right device plugged in?  The error dialog suggests you're selecting `100000kHz` as the speed, but your device only supports a maximum speed of `50000kHz`.

**Comment by graeme-hattan** [comment]
Yes, definitely, I only have one Segger device and it says J-Trace Pro on it!
